qutebrowser/utils/urlutils.py

```diff
@@ -196,17 +199,21 @@ def fuzzy_url(urlstr, cwd=None, relative=False, do_search=True):
     return url
 
 
-def _has_explicit_scheme(url):
+def _has_explicit_scheme(url, allow_only_scheme=True):
     """Check if an url has an explicit scheme given.
 
     Args:
         url: The URL as QUrl.
+        allow_only_scheme: if set to True the URL is allowed to contain only
+                           the scheme with an empty path.
+
     """
     # Note that generic URI syntax actually would allow a second colon
     # after the scheme delimiter. Since we don't know of any URIs
     # using this and want to support e.g. searching for scoped C++
     # symbols, we treat this as not an URI anyways.
     return (url.isValid() and url.scheme() and
+            (allow_only_scheme or len(url.path()) > 0) and
```

Comment:
- it makes sense
- ok. I'll make this the default and we will think about this if someone complains
- you're definetly right
- ok.